server/data/constants.ts

```diff
@@ -0,0 +1,20 @@
+export const transferPrisonId = 'TRN' as const
+export type TransferPrisonId = typeof transferPrisonId
+
+export const outsidePrisonId = 'OUT' as const
+export type OutsidePrisonId = typeof outsidePrisonId
+
+/**
+ * All user require role PRISON to access this site
+ */
+export const userRolePrison = 'ROLE_PRISON' as const
+
+/**
+ * Users without global search cannot search outside their active caseload nor link out to profiles outside their caseload set
+ */
+export const userRoleGlobalSearch = 'ROLE_GLOBAL_SEARCH' as const
+
+/**
+ * Users without inactive bookings cannot search for released prisoners nor link out to profiles of those individuals
+ */
+export const userRoleInactiveBookings = 'ROLE_INACTIVE_BOOKINGS' as const
```

Comment:
these 3 constants aren't needed in this pr cos these roles don't affect the access permissions.

what you could do (if you wanted) is add `ROLE_MAINTAIN_IEP` here and use it as a constant in place of that string elsewhere.

Reply:
Thanks - had brought these over but wasn't sure if i would need them down the line. Agree to adding `ROLE_MAINTAIN_IEP` here.